reference/src/layout/scalars.md

```diff
@@ -19,6 +19,15 @@ taking on the values 1 (`true`) or 0 (`false`).
 
 [Rust Layout and ABIs]: https://gankro.github.io/blah/rust-layouts-and-abis/#the-layoutsabis-of-builtins
 
+<details><summary><b>Rationale</b></summary> This allows directly using `bool` in the
```

Comment:
I would suggest moving "This..." to a new after `</summary>` which makes things clearer when reading the raw text.